e2e/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Copyright 2017 DigitalOcean
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM golang:1.12
+
+# Install kubectl
+RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.15.0/bin/linux/amd64/kubectl && \
+    chmod +x ./kubectl && \
+    mv ./kubectl /usr/local/bin/kubectl
+
+# Install kops
+RUN curl -Lo kops https://github.com/kubernetes/kops/releases/download/$(curl -s https://api.github.com/repos/kubernetes/kops/releases/latest | grep tag_name | cut -d '"' -f 4)/kops-linux-amd64 && \
```

Comment:
The latest kops release won't work for us until kubernetes/kops#7221 is merged and backported to the release versions. But even after that happens, the kops release won't contain the CCM manifest of the version under development.

I suppose the best we can do for now is check out the source code of kops in a version that works for us (1.10 right now, soon master once that upstream PR merges), change the built-in CCM manifest to point at something like `digitalocean/digitalocean-cloud-controller-manager:dev` (that would need to correspond to the version we want to test), compile, and use the built binary for our tests.

I'm completely okay with doing that in a follow-up PR -- the containerized end-to-end test execution is already a great step forward.

Reply:
Update: the kops PR was merged, we could use kops master now.